docs/topics/readme/rest-http-secured-README.adoc

```diff
@@ -95,16 +95,16 @@ $ curl -sk -X POST https://<SSO_AUTH_SERVER_URL>/auth/realms/$REALM/protocol/ope
   -d client_secret=$SECRET
 ----
 +
-NOTE: The `-sk` option tells curl to ignore failures resulting from self-signed certificates. Do not use this option in a production environment.
-
-. Extract the access token information using the `jq` tool:
+Optionally, you can extract the access token information, for example using the `jq` tool:
```

Comment:
Same as above, extracting the token is mandatory; what's optional is using the `jq` tool.